elements/pfe-code-block/demo/pfe-code-block.html

```diff
@@ -0,0 +1,39 @@
+<link rel="stylesheet" href="./demo.css"/>
+<script type="module" src="./pfe-code-block.js"></script>
+
+<article>
+  <h1>Code Block</h1>
+  <h2>Basic</h2>
+  <pfe-code-block id="code">
+    <span slot="code">apiVersion: helm.openshift.io/v1beta1/
```

Comment:
can we assume js is always on? if not, why not ape zero-md and use `<script type="{{mimetype}}">`?